fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/api/LogicalResourceIdentKey.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * (C) Copyright IBM Corp. 2020, 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.persistence.jdbc.dao.api;
+
+import java.util.Objects;
+
+/**
+ * A DTO representing a mapping of a logical_resource identity to its database
+ * logical_resource_id value.
+ */
+public class LogicalResourceIdentKey {
```

Comment:
in java 17, this could be a record :-)

i wonder if its worth marking such classes for eventual conversion...